crowbar_framework/app/models/service_object.rb

```diff
@@ -1478,6 +1463,58 @@ def wait_for_chef_daemons(node_list)
 
   private
 
+  def release_chef_locks(locks)
+    return if locks.empty?
+
+    locks.each_slice(20) do |lock_batch|
```

Comment:
Please can this `20` be changed to a constant?  I guess `THREAD_POOL_SIZE` or similar?